tests/phpunit/CRM/MembershipExtras/Job/OfflineAutoRenewal/MultiInstallmentPlanTest.php

```diff
@@ -761,4 +763,98 @@ private function getCustomFieldId($customGroupName, $customFieldName) {
     ]);
   }
 
+  public function testRenewalWithNonRenewableLineOnCurrentPeriodAndNewMembershipForNextPeriod() {
+    $paymentPlanMembershipOrder = new PaymentPlanMembershipOrder();
+    $paymentPlanMembershipOrder->membershipStartDate = date('Y-m-d', strtotime('-1 year -1 month'));
+    $paymentPlanMembershipOrder->paymentPlanFrequency = 'Monthly';
+    $paymentPlanMembershipOrder->paymentPlanStatus = 'Completed';
+    $paymentPlanMembershipOrder->lineItems[] = [
+      'entity_table' => 'civicrm_membership',
+      'price_field_id' => $this->testRollingMembershipTypePriceFieldValue['price_field_id'],
+      'price_field_value_id' => $this->testRollingMembershipTypePriceFieldValue['id'],
+      'label' => $this->testRollingMembershipType['name'],
+      'qty' => 1,
+      'unit_price' => $this->testRollingMembershipTypePriceFieldValue['amount'],
+      'line_total' => $this->testRollingMembershipTypePriceFieldValue['amount'],
+      'financial_type_id' => 'Member Dues',
+      'non_deductible_amount' => 0,
+      'auto_renew' => 0,
+    ];
+    $paymentPlan = PaymentPlanOrderFabricator::fabricate($paymentPlanMembershipOrder);
+    $this->addRenewableNewMembershipToNextPeriodOnly($paymentPlan, [
+      'name' => 'Test New Rolling Membership',
+      'period_type' => 'rolling',
+      'minimum_fee' => 1200,
+      'duration_interval' => 1,
+      'duration_unit' => 'year',
+    ]);
+
+    $multipleInstallmentRenewal = new MultipleInstallmentRenewalJob();
+    $multipleInstallmentRenewal->run();
+    $this->isPaymentPlanMembershipRenewed($paymentPlan['id'], '+1 year -1 month -1 day');
```

Comment:
this method only returns true or false, so calling it like that does not test for anything and instead you should assert the result of this method (see the other tests that call it), but I think the asserts below should be enough so I think it is fine to remove it complety.